pkg/controller/storagecluster/external_resources_test.go

```diff
@@ -385,3 +395,57 @@ func assertCephObjectStore(t *testing.T, reconciler ReconcileStorageCluster, rem
 		assert.Equal(t, hostFound, cObjS.Spec.Gateway.ExternalRgwEndpoints[0].IP)
 	}
 }
+
+func TestExternalResourceReconcile(t *testing.T) {
+	reconciler := createExternalClusterReconciler(t)
+	assertReconciliationOfExternalResource(t, reconciler)
+}
+
+func assertReconciliationOfExternalResource(t *testing.T, reconciler ReconcileStorageCluster) {
```

Comment:
Any reason to define this as a separate function? It doesn't look like it reduces cyclomatic complexity and is only ever called once.

Reply:
This is a common pattern we follow (at least in this file), that is; a `TestXXX` function which has multiple `assertXXX` function calls. As I mentioned, tests are not complete yet and are just written for the proof of concept (that reconciliation is working). It needs to be polished further, so will be changed.

Reply:
Okay, that's good enough!